src/Google.Bigquery.V2/TableSchemaBuilder.cs

```diff
@@ -38,133 +44,88 @@ public void Add(TableFieldSchema field)
         {
             Preconditions.CheckNotNull(field, nameof(field));
 
-            fields.Add(field);
+            _fields.Add(field);
         }
 
         /// <summary>
         /// Creates a field with the specified details, and adds it to the schema being built.
         /// </summary>
-        /// <param name="name">The name of the field. Must not be null.</param>
-        /// <param name="type">The type of the field. Must be a defined member within <see cref="SchemaFieldType"/>.</param>
-        /// <param name="mode">The mode of the field. Must be a defined member within <see cref="SchemaFieldMode"/>.</param>
+        /// <param name="name">The name of the field. Must not a valid field name.</param>
+        /// <param name="type">The type of the field. Must be a defined member within <see cref="BigqueryDbType"/>, other than <c>Record</c>.</param>
```

Comment:
I wondered about that. Was just drawing attention to the fact that it was validated. Happy to remove if you'd like though.

I've just spotted "Must not a valid field name" too...